src/Entity/TaskExecutionRepository.php

```diff
@@ -129,17 +129,13 @@ public function findByTaskUuid($taskUuid)
     /**
      * {@inheritdoc}
      */
-    public function findScheduled(\DateTime $dateTime = null)
```

Comment:
the interface has changed a few time ago. so i have removed it also from this class.